fail to run zkey-manager compile

Initially I got the error `/bin/sh: ../../.cargo/bin/circom: No such file or directory` so I changed my circomPath in `zkeys.config.yml` to point to my locally installed instance. Now when I run `yarn zkey-manager-compile` I get the following error:

```
> yarn zkey-manager-compile
yarn run v1.22.4
warning package.json: No license field
warning ../package.json: No license field
$ zkey-manager compile -c ./zkeys.config.yml
Compiling NftMint (prod) with params
template instances: 141
non-linear constraints: 453
linear constraints: 0
public inputs: 2
public outputs: 1
private inputs: 1
private outputs: 0
wires: 456
labels: 1352
Written successfully: ./zkeys/NftMint__prod.r1cs
Written successfully: ./zkeys/NftMint__prod.sym
Written successfully: ./zkeys/NftMint__prod_cpp/NftMint__prod.cpp and ./zkeys/NftMint__prod_cpp/NftMint__prod.dat
Written successfully: ./zkeys/NftMint__prod_cpp/main.cpp, circom.hpp, calcwit.hpp, calcwit.cpp, fr.hpp, fr.cpp, fr.asm and Makefile
Written successfully: ./zkeys/NftMint__prod_js/NftMint__prod.wat
Written successfully: ./zkeys/NftMint__prod_js/NftMint__prod.wasm
Everything went okay, circom safe
g++ -c main.cpp -std=c++11 -O3 -I.
main.cpp:9:10: fatal error: 'nlohmann/json.hpp' file not found
#include <nlohmann/json.hpp>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [main.o] Error 1
Error running the make command. Please check if all dependencies are present.
[String: 'g++ -c main.cpp -std=c++11 -O3 -I.\n'] {
  stdout: 'g++ -c main.cpp -std=c++11 -O3 -I.\n',
  stderr: "main.cpp:9:10: fatal error: 'nlohmann/json.hpp' file not found\n" +
    '#include <nlohmann/json.hpp>\n' +
    '         ^~~~~~~~~~~~~~~~~~~\n' +
    '1 error generated.\n' +
    'make: *** [main.o] Error 1\n',
  code: 2,
  cat: [Function: bound ],
  exec: [Function: bound ],
  grep: [Function: bound ],
  head: [Function: bound ],
  sed: [Function: bound ],
  sort: [Function: bound ],
  tail: [Function: bound ],
  to: [Function: bound ],
  toEnd: [Function: bound ],
  uniq: [Function: bound ]
}
main.cpp:9:10: fatal error: 'nlohmann/json.hpp' file not found
#include <nlohmann/json.hpp>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [main.o] Error 1

mv: no such file or directory: /Users/billy/GitHub/zk/zknftmint/circuits/zkeys/NftMint__prod_cpp/NftMint__prod
✨  Done in 3.61s.
```

Can you advise how to proceed?

--- a/README.md
+++ b/README.md
@@ -15,6 +15,8 @@ npm i
 npm run bootstrap
 ```
 
+You may need to manually install dependencies for `zkey-manager` by following instructions [here](https://github.com/privacy-scaling-explorations/zkey-manager#requirements).
+
 In a separate terminal, run a HTTP server in `web/zkeys`:
 
 ```bash

